script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_1.txt

```diff
@@ -80,31 +80,31 @@ Neither mine nor Arcueid's name is on the list.
 With a click, I turn off the TV.
 "―――"
 <Page25>
-The safety of the guests have yet to be confirmed―――?
-Not a single victim has yet to be discovered―――?
+The safety of the guests has yet to be confirmed―――?
+Not a single victim has been discovered―――?
 How can you say that when you have so much information?
 The information that all the people that stayed at the hotel were engulfed in those unfathomable flames and burned without a trace.
 <Page26>
 "――――"
-...I try to hold back my nausea.
+...I hold back my nausea.
 I can't think back to the tragedy or I'll end up vomiting.
-Receiving pity in this situation would make me worse than a pig.
+Wallowing in self-pity in this situation would make me worse than a pig.
 As the only one who escaped that hotel with his life intact, I cannot allow myself to feel anything but hatred.
 <Page27>
 Two hundred people.
-Two hundred human beings were slaughtered arbitrarily without being able to even retain their original form.
+Two hundred human beings were slaughtered arbitrarily, without even being allowed to retain their original form.
 <Page28>
-The face of that vampire seeps into my mind.
+The face of that vampire looms in my mind.
 I don't know what kind of person he is.
-All I know is that he's the source of this disaster. Without a doubt.
+But I do know, beyond a shadow of a doubt, that he is the cause of all this.
 ―――That's right.
 The only thing I know for certain is that he's someone who cannot be allowed to live.
 <Page29>
 "――――――"
 ...I wonder if my heart is still paralyzed.
 The fear continues to persist, even now.
 My rage continues to swirl intensely.
-And yet, my thoughts are becoming cold.
+And yet, my thoughts are completely cold.
```

Comment:
Hrm, I read it as "are in the state of *being* completely chilled" rather than "the state of *becoming* completely chilled". I feel like it makes sense for them to already be cold? If this is definitely the wrong jp interpretation though I'll change it

Reply:
weblio definition is " すっかり冷えてしまう" which pretty clearly reads to me as "become" with 冷えて. @yk1512 your take?

Reply:
Ben is right in this case yes. 冷え切っている refers to a process that is currently occurring. I think 冷え切っていた is what you would use when its completed.